### DISCUSSION TITLE
创建物化视图状态CANCELLED

### DISCUSSION THREAD

**Discussion by dqbrMiao** [apache/doris#31836] [A - General / Q&A]
对尝试对同一张表使用不同的<query>创建物化视图后通过SHOW ALTER TABLE MATERIALIZED VIEW查询任务状态，其中会存在部分CANCELLED状态的任务，错误信息为：errCode = 2, detailMessage = rollup tasks failed on same tablet reach threshold [[ALTER], signature: 38285, backendId: 32074, tablet id: 38285], reason=task type: ALTER, status_code: INTERNAL_ERROR, status_message: Don't support to add materialized view by linked schema change。翻阅物化视图相关文档没有发现linked schema change相关内容，求助下这是什么原因？

**Comment by yagagagaga** [comment]
What is the Doris version? What are the statements for table structure and materialized views?

**Reply by dqbrMiao** [reply]
**Doris version**:2.0.1
**CREATE TABLE SQL**:CREATE TABLE `mv_test` ( `EMPLOYEE_ID` int(11) NOT NULL COMMENT '序号', `EMPLOYEE_NAME` varchar(65533) NULL COMMENT '姓名', `DEPARTMENT` varchar(50) NULL COMMENT '部门', `POSITION` varchar(50) NULL COMMENT '岗位', `SALARY` DECIMAL(10, 2) NULL COMMENT '薪资', `PHONE_NUMBER` varchar(20) NULL COMMENT '电话', `ADDRESS` varchar(100) NULL COMMENT '地址' ) ENGINE=OLAP DUPLICATE KEY(`EMPLOYEE_ID`) COMMENT '明细模型' DISTRIBUTED BY HASH(`EMPLOYEE_ID`) BUCKETS 2 PROPERTIES ( "replication_allocation" = "tag.location.default: 1", "is_being_synced" = "false", "storage_format" = "V2", "light_schema_change" = "true", "disable_auto_compaction" = "false", "enable_single_replica_compaction" = "false" );
**CREATE MV SQL**:CREATE MATERIALIZED VIEW test_mv as select EMPLOYEE_ID from mv_test

查找了一下当时执行的创建mv的sql似乎是在编写的时候遗漏了一列聚合列，是否是因为这个原因？

**Comment by zhbdesign** [comment]
https://github.com/apache/doris/issues/15593
